components/cam/src/physics/crm/samxx/pressure.cpp

```diff
@@ -0,0 +1,294 @@
+
```

Comment:
Maybe we should add a comment here explaining the 2 FFT options?
Unless it's already explained somewhere else?

Reply:
I can add a comment for it. Once we've fully exercised the new FFTs, I can take this out. I had this in there to allow me to get double precision similarity with the standalone fortran code during development.